Tooda/Sources/Scenes/CreateNote/CreateNoteViewReactor.swift

```diff
@@ -296,7 +296,10 @@ extension CreateNoteViewReactor {
 extension CreateNoteViewReactor {
   private func registerButtonDidTapped() -> Observable<Mutation> {
     
-    self.dependency.coordinator.transition(to: .popUp(type: .list(self.addStickerCompletionRelay)), using: .modal, animated: false, completion: nil)
+    self.dependency.coordinator.transition(to: .popUp(type: .list(self.addStickerCompletionRelay)),
+                                           using: .modal,
+                                           animated: false,
+                                           completion: nil)
```

Comment:
가독성 생각하면 이 방식이 더 깔금하더라고요.
```suggestion
self.dependency.coordinator.transition(
     to: .popUp(type: .list(self.addStickerCompletionRelay)),
     using: .modal,
     animated: false,
     completion: nil
)
```